### ISSUE TITLE
ply Point Size not changing with Slider

### ISSUE THREAD

**Issue by Aupuma** [cnr-isti-vclab/meshlab#1403]
- Platform: Windows 11
 - Version: 2022.02 Installer
When I open a .ply file with a point cloud and try modifying the point size, nothing happens. However, If I select vertices and modify the Point Size, I can see the red dots growing and shrinking

**Comment by alemuntoni** [comment]
Could you please share the ply file?

**Comment by Aupuma** [comment]
Yes, @alemuntoni here is the file I've used. I've also made a screen recording showing you that the selection points change size but not the source points
[83613-CoworkingAgaete1-sparse.zip](https://github.com/cnr-isti-vclab/meshlab/files/12038084/83613-CoworkingAgaete1-sparse.zip)

https://github.com/cnr-isti-vclab/meshlab/assets/22346231/234b3bbc-c3c4-4380-a64b-1334c013d24f

**Comment by adizhol-str** [comment]
I have a similar problem, where some .ply files load with a bigger point size and don't change after modifiying the slider.
Other .ply  point clouds load just fine
